Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateWebGLRenderingContext.cpp

```diff
@@ -890,14 +890,16 @@ class @class_name@ {
 
             if (webgl_version == 2) {
                 function_impl_generator.append(R"~~~(
+    if (src_offset + src_length > count) {
```

Comment:
this check still does not seem to be correct because src_offset and src_length are offset and length of a vector while count is number of matrices in this vector.